src/website/app/demos/Radio/examples/Radio/importSyntax.js

```diff
@@ -7,11 +7,13 @@ import Radio from 'mineral-ui/Radio';
 \`\`\`
 
 <Callout title="Note">
-  Mineral UI also provides a <a href="../radio-group" key={0}>RadioGroup</a>,
-  which offers a simpler API for working with a group of Radios, and
-  a <a href="../form-field" key={1}>FormField</a> to provide an accessible label
-  and other features. Examples here omit RadioGroups and FormFields for clarity
-  and brevity.
+  <p key={0}>
```

Comment:
I don't understand the `<p key={0}>` thing. how does this help / what problem does this solve?

Reply:
Two details here:

1. Our third-party markdown parser requires keys when using JSX as children like this. Not sure why, honestly.
2. We can't just wrap all Callout in a `p` because some Callouts contain multiple `p`s and nesting a `p` inside another is invalid HTML.